moesifapi/parse_body.py

```diff
@@ -24,6 +24,24 @@ def transform_headers(cls, headers):
     def base64_body(cls, body):
         return base64.standard_b64encode(body).decode(encoding="UTF-8"), "base64"
 
+    @staticmethod
+    def get_content_type(headers):
+        try:
+            return headers['content-type']
+        except KeyError:
+            try:
+                return headers['Content-Type']
+            except KeyError:
+                pass
+        return None
+
+    def is_multi_part_upload(self, headers):
```

Comment:
@keyur9 , is this used? I dont see we dropping payload when true

Reply:
@dgilling this is base api, we'd used this function in the middleware where we'd be dropping if it's multipart payload